### PR TITLE
365 vat python testing additions

### PR DESCRIPTION
**Pull request by RoriCremer**
Docker image added---I think we'll want to implement whatever we are doing for the jars for the docker images in parallel

I'm planning to add an additional python test to this and then I want to dig into how to make sure they run during docker build

outside the scope for this specific round of python testing, but it will be worth reaching out the the coworker who spotted both previous clinvar issues to ask for additional spot testing cases

### CODE REVIEW

**Review by rsasch — CHANGES_REQUESTED** [review]
I tried running ./build_docker.sh and it failed with the following error:
```
ImportError: Failed to import test module: create_variant_annotation_table_test
Traceback (most recent call last):
  File "/opt/miniconda3/envs/gvs/lib/python3.8/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/Users/rasch/p/gatk/scripts/variantstore/wdl/extract/create_variant_annotation_table_test.py", line 13, in <module>
    from create_variant_annotation_table import make_annotated_json_row
  File "/Users/rasch/p/gatk/scripts/variantstore/wdl/extract/create_variant_annotation_table.py", line 8, in <module>
    import ijson
ModuleNotFoundError: No module named 'ijson'
```

File: scripts/variantstore/wdl/GvsCreateVATFromAnnotations.wdl [hunk 1]
```diff
@@ -24,14 +24,14 @@ workflow GvsCreateVATFromAnnotations {
...
-            service_account_json_path = service_account_json_path
-        }
+            ## Parse the annotations
+            call PrepAnnotationJson {
+              input:
+                annotation_json = GetAnnotations.input_jsons[i],
+                output_file_suffix = basename(GetAnnotations.input_jsons[i], "_annotated.json.gz") + ".json.gz",
+                output_path = output_path,
+                service_account_json_path = service_account_json_path
+            }
```
[the remark this answers is not in the document]
Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsCreateVATFromAnnotations.wdl:34]: I think it's just copy pasta? from a rebase annoyingly

**Comment by RoriCremer** [comment]
oooh, re: ijson, should I just add checking for and installing it into the ./build_docker.sh file?

I've had it forever since I need it to run the python script. It's in requirements.txt, where else should I put it?

**Comment by rsasch** [comment]
I think it would be good for there to be as little lift as possible required to run the tests, so adding it to ./build_docker.sh is probably best.

**Review by mcovarr — CHANGES_REQUESTED** [review]
a couple of Python nitpicks, otherwise looks good

**Comment by RoriCremer** [comment]
Bec's change resolved!

**Pull request merged**